### DISCUSSION TITLE
润加拿大相关

### DISCUSSION THREAD

**Discussion by Mega-MU** [The-Run-Philosophy-Organization/run#458] [Q&A]
这里有朋友是在加拿大的吗？我现在是准备润加拿大，当前是现在是自动化专业大一（下个学期就分专业到机械了），准备去加拿大上大学，有什么需要注意的事项吗？
顺便，OUAC和一些其他的学校的申请资料填写有人可以指点一下吗？

**Comment by Mapleglan** [comment]
我在加拿大，可以给你提供些帮助或建议。
